index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div>Заголовок 1 уровня</div>
```

Comment:
Здесь лучше использовать h1 тег

```suggestion
  <h1>Заголовок 1 уровня</h1>
```